lib/synapse/haproxy.rb

```diff
@@ -386,6 +388,7 @@ class Haproxy
         "option accept-invalid-http-request",
         "option accept-invalid-http-response",
         "option allbackups",
+        "option allredisp",
```

Comment:
Heh, this isn't even the option that I ended up getting accepted. http://marc.info/?l=haproxy&m=143161966505891&w=2 basically added a parameter to "option redispatch" which allows this behavior. Until that is released in 1.6 do you mind if we keep this in here (we use is pretty heavily at Yelp)